neofetch

```diff
@@ -1008,6 +1008,14 @@ get_distro() {
                         distro="Proxmox VE ${distro%/*}"
                 esac
 
+            elif type -p twistver >/dev/null; then
+                . /etc/os-release
+                case $distro_shorthand in
+                    on)   distro="$( twistver ) (${NAME} ${VERSION_ID})" ;;
+                    tiny) distro="$( twistver ) (${NAME})" ;;
+                    off)  distro="$( twistver ) (${PRETTY_NAME}})" ;;
+                esac
+
```

Comment:
@dylanaraps

Reply:
I am happy to merge this PR without the OS detection code. Your distribution should properly identify itself
otherwise neofetch must support every "pseudo-distribution" which makes things chaotic.

Reply:
https://github.com/dylanaraps/neofetch/blob/b3e21d89cd4acb09cc89b1ae4aca0929111176c7/neofetch#L974-L1009


But all these distros do use a custom code to detect distro including armbian

Reply:
> Your distribution should properly identify itself

It does identify.

Twister OS has its distro info in the command `twistver`

Reply:
@dylanaraps ^

Reply:
@dylanaraps

Reply:
Your distribution should identify itself via /etc/os-release or lsb-release. Exceptions were made in the past however I am reluctant to make any more. I will not merge this with the distribution detection code attached.

Reply:
Okay